### DISCUSSION TITLE
I'm getting this error

### DISCUSSION THREAD

**Discussion by narf12-00** [mifi/lossless-cut#2008] [Support]
Error: Command failed with exit code 1: /Applications/LosslessCut.app/Contents/Resources/ffmpeg -hide_banner -ss 544.15782 -i /Users/francescomancin/Desktop/UNIVERSITÀ/CHIRURGIA/Audio Chirurgia/VINCI/BIFOSFONATI.m4a -t 195.08758 -avoid_negative_ts auto -map 0:0 -c:0 copy -map_metadata 0 -default_mode infer_no_subs -ignore_unknown -f mp3 -y /Users/francescomancin/Desktop/UNIVERSITÀ/CHIRURGIA/Audio Chirurgia/VINCI/test/1$INTRODUZIONE_BIFOSFONATI_-specializzanda.mp3 
Input #0, mov,mp4,m4a,3gp,3g2,mj2, from '/Users/francescomancin/Desktop/UNIVERSITÀ/CHIRURGIA/Audio Chirurgia/VINCI/BIFOSFONATI.m4a':
  Metadata:
    major_brand     : M4A 
    minor_version   : 0
    compatible_brands: M4A isommp42
    creation_time   : 2024-03-18T15:46:17.000000Z
    title           : BIFOSFONATI
    voice-memo-uuid : A003D34C-73B1-4D72-AB68-DE601E9FCF64
    encoder         : com.apple.VoiceMemos (iPad Versione 14.4.1 (Build 23E224))
  Duration: 01:19:07.33, start: 0.000000, bitrate: 67 kb/s
  Stream #0:0[0x1](und): Audio: aac (LC) (mp4a / 0x6134706D), 48000 Hz, mono, fltp, 62 kb/s (default)
    Metadata:
      creation_time   : 2024-03-18T15:46:17.000000Z
      handler_name    : Core Media Audio
      vendor_id       : [0][0][0][0]
[mp3 @ 0x1251049e0] Invalid audio stream. Exactly one MP3 audio stream is required.
[out#0/mp3 @ 0x125104920] Could not write header (incorrect codec parameters ?): Invalid argument
[aost#0:0/copy @ 0x125105570] Error initializing output stream: 
Stream mapping:
  Stream #0:0 -> #0:0 (copy)
    Last message repeated 1 times
    at makeError (/Applications/LosslessCut.app/Contents/Resources/app.asar/node_modules/execa/lib/error.js:60:11)
    at handlePromise (/Applications/LosslessCut.app/Contents/Resources/app.asar/node_modules/execa/index.js:118:26)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

{
  "err": {
    "killed": false,
    "failed": true,
    "timedOut": false,
    "isCanceled": false,
    "exitCode": 1
  },
  "state": {
    "ffmpegExperimental": false,
    "preserveMovData": false,
    "movFastStart": false,
    "preserveMetadataOnMerge": false,
    "filePath": "/Users/francescomancin/Desktop/UNIVERSITÀ/CHIRURGIA/Audio Chirurgia/VINCI/BIFOSFONATI.m4a",
    "fileFormat": "mp3",
    "externalFilesMeta": {},
    "mainStreams": [
      {
        "index": 0,
        "codec_name": "aac",
        "codec_long_name": "AAC (Advanced Audio Coding)",
        "profile": "LC",
        "codec_type": "audio",
        "codec_tag_string": "mp4a",
        "codec_tag": "0x6134706d",
        "sample_fmt": "fltp",
        "sample_rate": "48000",
        "channels": 1,
        "channel_layout": "mono",
        "bits_per_sample": 0,
        "initial_padding": 0,
        "id": "0x1",
        "r_frame_rate": "0/0",
        "avg_frame_rate": "0/0",
        "time_base": "1/48000",
        "start_pts": 0,
        "start_time": "0.000000",
        "duration_ts": 227871744,
        "duration": "4747.328000",
        "bit_rate": "62861",
        "nb_frames": "222531",
        "extradata_size": 2,
        "disposition": {
          "default": 1,
          "dub": 0,
          "original": 0,
          "comment": 0,
          "lyrics": 0,
          "karaoke": 0,
          "forced": 0,
          "hearing_impaired": 0,
          "visual_impaired": 0,
          "clean_effects": 0,
          "attached_pic": 0,
          "timed_thumbnails": 0,
          "captions": 0,
          "descriptions": 0,
          "metadata": 0,
          "dependent": 0,
          "still_image": 0
        },
        "tags": {
          "creation_time": "2024-03-18T15:46:17.000000Z",
          "language": "und",
          "handler_name": "Core Media Audio",
          "vendor_id": "[0][0][0][0]"
        }
      }
    ],
    "copyStreamIdsByFile": {
      "/Users/francescomancin/Desktop/UNIVERSITÀ/CHIRURGIA/Audio Chirurgia/VINCI/BIFOSFONATI.m4a": {
        "0": true
      }
    },
    "cutSegments": [
      {
        "end": 544.157815
      },
      {
        "start": 544.157815,
        "end": 739.2454
      },
      {
        "start": 739.2454,
        "end": 820.321987
      },
      {
        "start": 820.321987,
        "end": 992.189734
      },
      {
        "start": 992.189734,
        "end": 1188.731203
      },
      {
        "start": 1188.731203,
        "end": 1255.00556
      },
      {
        "start": 1255.00556,
        "end": 1333.56196
      },
      {
        "start": 1333.56196,
        "end": 1402.044151
      },
      {
        "start": 1402.044151,
        "end": 1505.705523
      },
      {
        "start": 1505.705523,
        "end": 1693.984068
      },
      {
        "start": 1693.984068,
        "end": 1836.721559
      },
      {
        "start": 1836.721559,
        "end": 1945.733695
      },
      {
        "start": 1945.733695,
        "end": 2153.042331
      },
      {
        "start": 2153.042331,
        "end": 2279.706466666667
      },
      {
        "start": 2279.706466666667,
        "end": 2312.74803
      },
      {
        "start": 2312.74803,
        "end": 2620.206826
      },
      {
        "start": 2620.206826,
        "end": 2643.404428
      },
      {
        "start": 2643.404428,
        "end": 2728.628106
      },
      {
        "start": 2728.628106,
        "end": 2758.283674
      },
      {
        "start": 2758.283674,
        "end": 3260.539372
      },
      {
        "start": 3260.539372,
        "end": 3366.606714
      },
      {
        "start": 3366.606714,
        "end": 4121.909117
      },
      {
        "start": 4121.909117,
        "end": 4128.931924
      },
      {
        "start": 4128.931924,
        "end": 4170.812714
      },
      {
        "start": 4170.812714
      }
    ],
    "mainFileFormatData": {
      "filename": "/Users/francescomancin/Desktop/UNIVERSITÀ/CHIRURGIA/Audio Chirurgia/VINCI/BIFOSFONATI.m4a",
      "nb_streams": 1,
      "nb_programs": 0,
      "format_name": "mov,mp4,m4a,3gp,3g2,mj2",
      "format_long_name": "QuickTime / MOV",
      "start_time": "0.000000",
      "duration": "4747.328000",
      "size": "40225233",
      "bit_rate": "67785",
      "probe_score": 100,
      "tags": {
        "major_brand": "M4A ",
        "minor_version": "0",
        "compatible_brands": "M4A isommp42",
        "creation_time": "2024-03-18T15:46:17.000000Z",
        "title": "BIFOSFONATI",
        "voice-memo-uuid": "A003D34C-73B1-4D72-AB68-DE601E9FCF64",
        "encoder": "com.apple.VoiceMemos (iPad Versione 14.4.1 (Build 23E224))"
      }
    },
    "rotation": 360,
    "shortestFlag": false,
    "effectiveExportMode": "separate",
    "outSegTemplate": "${SEG_NUM}$${SEG_LABEL}${EXT} "
  },
  "platform": "darwin",
  "version": "3.59.1"
}

**Comment by mifi** [comment]
set output format to mp4/ipod (not mp3)
